File: src/main/java/io/openfuture/openmessanger/service/response/UserResponse.java
package io.openfuture.openmessanger.service.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserResponse {
    String id;
    String firstName;
    String lastName;
    String email;
}

File: src/main/kotlin/io/openfuture/openmessanger/service/response/UserResponse.kt
package io.openfuture.openmessanger.service.response

data class UserResponse(
    var id: String? = null,
    var firstName: String? = null,
    var lastName: String? = null,
    var email: String? = null,
    var orgId: String? = null,
    var modifiedTime: String? = null,
    var role: String? = null
)